infrastructure/tf_backend/variables.tf

```diff
@@ -2,7 +2,7 @@ variable "project_name" {
   type = string
 
   validation {
-    condition     = var.project_name != "<fill-project-name>"
+    condition     = var.project_name != ""
```

Comment:
I think we should be checking for empty values, not some placeholder values, since our `tf_backend.auto.tfvars` is prefilled with a potential value (like `aaq`).

Reply:
That makes sense.